synapseclient/core/utils.py

```diff
@@ -1015,8 +1016,12 @@ def printTransferProgress(
         postfix,
         status,
     )
-    sys.stdout.write(text)
-    sys.stdout.flush()
+    # Used for backwards compatability if anyone happens to be using this function
+    if logger:
+        logger.info(text)
+    else:
+        sys.stdout.write(text)
+        sys.stdout.flush()
```

Comment:
This is to make sure that the tqdm bar is playing nicely with these messages that get printed out for table operations. Internal to the "new" interface, we are still using some of the query functionality from the "old" client. Example of the console:
![image](https://github.com/user-attachments/assets/13b52772-3a26-4693-9794-e99ac0bad197)

(It was much worse before :D )